_releaser/aws.go

```diff
@@ -100,15 +100,15 @@ type AwsCloudfrontUpdateCmd struct {
 	Function              string `kong:"name='lambda-function',env='AWS_LAMBDA_FUNCTION'"`
 	FunctionFile          string `kong:"name='lambda-function-file',env='AWS_LAMBDA_FUNCTION_FILE'"`
 	CloudfrontID          string `kong:"name='cloudfront-id',env='AWS_CLOUDFRONT_ID'"`
-	RedirectsJSON         string `kong:"name='redirects-json',env='REDIRECTS_JSON'"`
-	RedirectsPrefixesJSON string `kong:"name='redirects-prefixes-json',env='REDIRECTS_PREFIXES_JSON'"`
+	RedirectsFile         string `kong:"name='redirects-json',env='REDIRECTS_FILE'"`
```

Comment:
I think there is `file` type with `kong` to check file existence, let me check that

Reply:
I checked here but couldn't see it: https://github.com/alecthomas/kong?tab=readme-ov-file#supported-tags

Reply:
Oh maybe `type` https://github.com/alecthomas/kong?tab=readme-ov-file#custom-named-decoders